packages/server/controllers/events.js

```diff
@@ -135,6 +135,17 @@ async function getUpcomingEvents(currentDate, guildId) {
   return upcomingEvents;
 }
 
+async function getArchivedEvents(currDate, pastDate, guildId) {
+  const archivedEvents = await prisma.events.findMany({
+    where: {
+      guildId: guildId,
+      AND: [{ startDate: { gte: pastDate } }, { endDate: { lte: currDate } }],
+    },
+  });
```

Comment:
Let's also sort the archived events from most recent to least recent, so the events that happened most recently to the oldest events.